src/renderer/fonts/font_options.rs

```diff
@@ -95,6 +95,29 @@ impl PartialEq for FontOptions {
     }
 }
 
+fn parse_font_name(font_name: impl Into<String>) -> String {
+    let font_name: String = font_name.into();
+    let mut char_iter = font_name.chars().into_iter();
+
+    let mut parsed_font_name = String::with_capacity(font_name.len());
+
+    while let Some(current_char) = char_iter.next() {
+        let result_char = match current_char {
+            '\\' => char_iter.next(),
+            '_' => Some(' '),
+            _ => Some(current_char),
+        };
+
+        if let Some(result_char) = result_char {
+            parsed_font_name.push(result_char);
+        } else {
+            break;
+        }
+    }
```

Comment:
This loop is written in a slightly confusing way, imo. Instead of pushing char for char to the string, I'd guess declaring `itertools` as an explicit dependency (which is already in our dep tree anyways, see `cargo tree --invert itertools`) and using `Itertools::batching` would be worth a thought:

```suggestion
    let mut parsed_font_name = char_iter
        .chars()
        .batching(|iter| match iter.next()? {
            '\\' => iter.next(),
            '_' => Some(' '),
            ch => Some(ch),
        })
        .collect();
```